### PR TITLE
(gh-495) Implements Pre-Logon Access Provider for OpenVPN

### LINKED ISSUE #495: Add OpenVPN parameter for Pre-Logon Access Provider
OpenVPN has a [Pre-Logon Access Provider](https://github.com/OpenVPN/openvpn-gui/issues/77) since version 2.6.0 (I guess).

The function is activated with the `OpenVPN.PLAP.Register` option for the msi-installation.

### PR DESCRIPTION
**Pull request by anicoa**
Should resolve #495.

`chocolateyInstall.ps1`, `README.md` and `openvpn.nuspec` have been updated accordingly.

Has been tested with `choco install openvpn --package-parameters="/guionlogon /service /dcodriver /gui /addtodesktop /PLAP /WintunDriver" -s ." 

First PR for chocolatey, if anything is wrong I appreciacte feedback ;)

### CODE REVIEW

File: automatic/openvpn/README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ following package parameter can be set:
 * `/EasyRsa`          - install EasyRSA 3 scripts for X509 certificate management
 * `/DcoDriver`        - install the OpenVPN Data Channel Offload driver
 * `/TapDriver`        - install the TAP-Windows driver (NDIS6)
+* `/PLAP`             - install Pre-Logon Access-Provider
```
Comment (dgalbraith) [inline automatic/openvpn/README.md:45]: Placement of the ```/PLAP``` parameter should be consistent between the ```README.md``` and the ```openvpn.nuspec``` file.  At present the order in ```README.md``` is TapDriver --> PLAP --> WintunDriver versus the order in ```openvpn.nuspec``` of WintunDriver --> PLAP --> Documentation.

Please ensure the placement of the ```PLAP``` parameter is consistent across the 2 files.

Given the updates to the install script I would suggest perhaps WintunDriver --> PLAP --> Documentation which would be consistent ordering across all files.

File: automatic/openvpn/openvpn.nuspec [hunk 2]
```diff
@@ -57,6 +57,7 @@ following package parameter can be set:
 * `/DcoDriver`        - install the OpenVPN Data Channel Offload driver
 * `/TapDriver`        - install the TAP-Windows driver (NDIS6)
 * `/WintunDriver`     - install the layer 3 TUN driver for Windows
+* `/PLAP`             - instakl PRE-Logon Access-Provider
```
Comment (dgalbraith) [inline automatic/openvpn/openvpn.nuspec:60]: Please correct the spelling on the action instakl --> install and update to use consistent capitalization for PRE - other usage is Pre.

**Review by dgalbraith — CHANGES_REQUESTED** [review]
Thanks for the contributions @anicoa - appreciated!

If you could take a look at the comments I have made it would be fantastic - just a few small observations to ensure consistency across the updates if that is ok.

**Comment by anicoa** [comment]
thanks a lot for the fast feedback. I changed it accordingly

**Comment by dgalbraith** [comment]
@anicoa all merged now - thanks for the contribution.

The changes will be deployed to the Community repository the next time there is an update to the OpenVPN version at source.  When this occurs the new handling for PLAP will be picked up and deployed with the next build.

**Pull request merged**